test/unit/io.js

```diff
@@ -861,6 +861,54 @@ describe('Input/output', function () {
     });
   });
 
+  it('TIFF lzw compression does not throw error', function () {
```

Comment:
I guess we can test that the file got smaller for the testing image, I'll add that.